anubia_support/__openerp__.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#
+#    Copyright (c) All rights reserved:
+#        (c) 2014      Anubía, soluciones en la nube,SL (http://www.anubia.es)
+#                      Jorge Soto García <sotogarcia@gmail.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses
+#
+##############################################################################
+
+{
+    'name': "Anubia Support",
+    'version': '0.1',
+
+    'summary': """Anubia, technical support module""",
+
+    'description': """
```

Comment:
Description here is not needed if there exists a README.rst (.rst, not .md, as Odoo does not support .md). That is what OCA uses right now (take a look at https://github.com/OCA/maintainer-tools/tree/master/template/module ), so enhanced description can be done, and way easier than HTML description (which is also in OCA templates). We could go that route.